### ISSUE TITLE
Fix `delete --help` output

### ISSUE THREAD

**Issue by onemorepereira** [weaveworks/eksctl#1529]
**What happened?**

Issuing the command `eksctl delete --help` returns the following...

```bash
$ eksctl delete --help
Delete resource(s)

Usage: eksctl delete [flags]

Commands:
  eksctl delete cluster                 Delete a cluster
  eksctl delete nodegroup               Delete a nodegroup
  eksctl delete iamserviceaccount       Create an iamserviceaccount - AWS IAM role bound to a Kubernetes service account
  eksctl delete iamidentitymapping      Delete a IAM identity mapping

Common flags:
  -C, --color string   toggle colorized logs (valid options: true, false, fabulous) (default "true")
  -h, --help           help for this command
  -v, --verbose int    set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging (default 3)

Use 'eksctl delete [command] --help' for more information about a command.
```

**What you expected to happen?**

Expecting the help text for the `iamserviceaccount` delete option to read something like this instead:

```bash
eksctl delete iamserviceaccount       Delete an IAM service account
```

**How to reproduce it?**

Run...
```bash
$ eksctl delete --help
```

**Anything else we need to know?**
N/A

**Versions**

```bash
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T19:18:23Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
The connection to the server localhost:8080 was refused - did you specify the right host or port?
$ eksctl version
[ℹ]  version.Info{BuiltAt:"", GitCommit:"", GitTag:"0.8.0"}
```

**Logs**

```bash
$ eksctl delete --help
Delete resource(s)

Usage: eksctl delete [flags]

Commands:
  eksctl delete cluster                 Delete a cluster
  eksctl delete nodegroup               Delete a nodegroup
  eksctl delete iamserviceaccount       Create an iamserviceaccount - AWS IAM role bound to a Kubernetes service account
  eksctl delete iamidentitymapping      Delete a IAM identity mapping

Common flags:
  -C, --color string   toggle colorized logs (valid options: true, false, fabulous) (default "true")
  -h, --help           help for this command
  -v, --verbose int    set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging (default 3)

Use 'eksctl delete [command] --help' for more information about a command.
```

**Comment by onemorepereira** [comment]
I have a PR submitted addressing this simple issue: #1527

**Comment by onemorepereira** [comment]
Resolved under #1527
